tests/integration/mutation/create/constraints/size_constraint_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package constraints
+
+import (
+	"testing"
+
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestMutationCreate_WithSizeConstrain_ShouldSucceed(t *testing.T) {
+	test := testUtils.TestCase{
+		Description: "Simple create mutation with size contraint",
+		Actions: []any{
+			testUtils.SchemaUpdate{
+				Schema: `
+					type Users {
+						name: String
+						numbers: [Int!] @constraints(size: 2)
+					}
+				`,
+			},
+			testUtils.CreateDoc{
+				Doc: `{
+					"name": "John",
+					"numbers": [27, 28]
+				}`,
+			},
+			testUtils.Request{
+				Request: `
+					query {
+						Users {
+							_docID
+							name
+							numbers
+						}
+					}
+				`,
+				Results: map[string]any{
+					"Users": []map[string]any{
+						{
+							"_docID":  testUtils.NewDocIndex(0, 0),
```

Comment:
suggestion: `_docID` is quite an expensive property maintenance-wise to include in a test, and it does not appear to have a use here, I suggest removing it from the query.

Same goes for the p[n]counter tests.

Reply:
Done